src/components/form/form-test.stories.tsx

```diff
@@ -367,3 +370,82 @@ export const DefaultWithPager = () => (
 );
 
 DefaultWithPager.storyName = "default with pager";
+
+export const MockFormForAriaLiveDemo = () => {
+  const [textareaValue, setTextareaValue] = useState("");
+  const [textboxValue, setTextboxValue] = useState("");
+  const [textEditorValue, setTextEditorValue] = useState(
+    EditorState.createEmpty()
+  );
+  const [passwordValue, setPasswordValue] = useState("");
+
+  const resetValues = (event: { preventDefault: () => void }) => {
+    event.preventDefault(); // prevents the page from changing
+    setTextareaValue("");
+    setTextboxValue("");
+    setTextEditorValue(EditorState.createEmpty());
+    setPasswordValue("");
+  };
+
+  return (
+    <Form
+      fieldSpacing={0}
+      onSubmit={() => console.log("submit")}
```

Comment:
**suggestion(non-blocking)**: since we want to enact the reset on form submission, what do you think about calling `resetValues` within the `onSubmit` event handler so this is clear?

Reply:
Yeah I can do that. The idea behind this was that it is purely for manual testing. I just wanted something to let QA's know that `onSubmit` had been fired.